### PR TITLE
English review of #2 Hands On

### PR DESCRIPTION
**Pull request by Crissman**
Correct English spellings
Fix for non-NVIDIA GPU computers to use gpu_id = -1
Remove unnecessary to_gpu and to_cpu code
Reorder lists of extensions explanations to match code
Make all code words use code formatting with `code` style instead of 'code'.
Minor text fixes

** Please test on NVIDIA GPU machine to ensure new code works!

If this is ok, I'll make the similar ordering and code changes to the JPY version as well.

### CODE REVIEW

File: 2-Try-Trainer-class_en.ipynb [hunk 1]
```diff
@@ -104,19 +104,20 @@
     "        h2 = F.relu(self.l2(h1))\n",
     "        return self.l3(h2)\n",
     "\n",
-    "gpu_id = 0\n",
+    "gpu_id = 0  # Set to -1 if you don't have a GPU\n",
     "\n",
     "model = MLP()\n",
-    "model.to_gpu(gpu_id)"
+    "if gpu_id >= 0:\n",
+    "        model.to_gpu(gpu_id)"
```
Comment (mitmul) [inline 2-Try-Trainer-class_en.ipynb]: Could you use 4 spaces as 1 tab? There are 8 spaces before `model.to_gpu(gpu_id)`. 4 spaces are better.

File: 2-Try-Trainer-class_en.ipynb [hunk 2]
```diff
@@ -147,26 +148,25 @@
...
-    "#'softmax_cross_entropy' is used.\n",
+    "# 'softmax_cross_entropy' is used.\n",
     "# The output of this modified 'model' will now be a loss value instead\n",
     "# of a class label prediction.\n",
     "model = L.Classifier(model)\n",
     "\n",
     "# Send the model to the GPU.\n",
-    "model.to_gpu(gpu_id)\n",
+    "if gpu_id >= 0:\n",
+    "        model.to_gpu(gpu_id)\n",
```
Comment (mitmul) [inline 2-Try-Trainer-class_en.ipynb]: Ditto.

File: 2-Try-Trainer-class_en.ipynb [hunk 3]
```diff
@@ -450,13 +450,12 @@
...
     "from chainer import serializers\n",
-    "from chainer.cuda import to_gpu\n",
-    "from chainer.cuda import to_cpu\n",
     "\n",
     "model = MLP()\n",
     "serializers.load_npz('mnist_result/model_epoch-10', model)\n",
-    "model.to_gpu(gpu_id)\n",
     "\n",
+    "if gpu_id >=0:\n",
+    "        model.to_gpu(gpu_id)\n",
```
Comment (mitmul) [inline 2-Try-Trainer-class_en.ipynb]: Ditto

**Review by mitmul — CHANGES_REQUESTED** [review]
Thank you for revising the notebook. LGTM except the incorrect number of spaces for a tab.

**Review by mitmul — APPROVED** [review]

**Comment by Crissman** [comment]
Done.

**Pull request merged**